lang/macro/src/ir/data.rs

```diff
@@ -24,6 +24,7 @@ use quote::ToTokens;
 use syn::{
     punctuated::Punctuated,
     spanned::Spanned as _,
+    ItemFn,
```

Comment:
In the other codebases we normally do not directly import `syn` items and instead use their short `syn` prefix to make it clear in the code that these are `syn` types. This is especially useful when ink! shares a lot of common names. Not particularly useful for this small macro though. So `ir::` is always from ink! and `syn::` is always from ... well `syn`.